deploy/stacks/backend_stack.py

```diff
@@ -309,6 +318,23 @@ def __init__(
                 resource_prefix=resource_prefix,
                 vpc=vpc,
                 logs_bucket=s3_resources_stack.logs_bucket,
-                cw_alarm_action=monitoring_stack.cw_alarm_action,
+                cw_alarm_action=self.monitoring_stack.cw_alarm_action,
                 internet_facing=internet_facing,
             )
+
+    def create_opensearch_stack(self):
+        os_stack = OpenSearchStack(self, 'OpenSearch', **self.opensearch_args)
+        self.set_es_alarms(
+            alarm_name=f'{self.resource_prefix}-{self.envname}-opensearch-alarm',
+            domain_name=os_stack.domain_name,
+            cw_alarm_action=self.monitoring_stack.cw_alarm_action,
+        )
+
+    def create_opensearch_serverless_stack(self):
+        aoss_stack = OpenSearchServerlessStack(self, 'OpenSearchServerless', **self.opensearch_args)
+        self.monitoring_stack.set_aoss_alarms(
+            alarm_name=f'{self.resource_prefix}-{self.envname}-opensearch-serverless-alarm',
+            collection_id=aoss_stack.collection_id,
+            collection_name=aoss_stack.collection_name,
+            cw_alarm_action=self.monitoring_stack.cw_alarm_action,
```

Comment:
Super minor: looks like `set_aoss_alarms` is called only from this place and we don't need `set_aoss_alarms` to have a parameter `cw_alarm_action` since we can use it directly from `monitoring_stack`

Reply:
Thanks, resolved